server/plugin.go

```diff
@@ -41,6 +41,12 @@ const (
 	RootURLKey = "root_url"
 )
 
+// ILogger is used for logging in the plugin
+type ILogger interface {
+	LogWarn(msg string, keyValuePairs ...interface{})
+	LogInfo(msg string, keyValuePairs ...interface{})
+}
```

Comment:
0/5 the logger should have `LogDebug`, `LogInfo`, `LogWarn`, and `LogError` even if they are not used. Mainly because that's what our programs can expect from the logger. Non-blocking request to add the others